client/src/pages/UserProfile.js

```diff
@@ -0,0 +1,234 @@
+  },
+  // Footer
+  mapImage: {
+    width: "100%",
+  },
+  Typography: {
+    fontFamily: "Montserrat, sans-serif",
+  },
+}));
+
+export default function ProfilePage(props) {
+  const classes = useStyles();
+  const bioTitle = "Hi everyone!";
+  const bio = "I'm a foodie and I love to eat healthy tasty meals. Also I'm a mom of two kids";
+  return (
+    <Grid container xs={12} className={classes.outerGrid}>
+      <Grid container xs={12} lg={8} className={classes.innerGrid}>
+        {/* heading container */}
+        <Grid container xs={12} className={classes.cardHeader}>
+          <Grid item lg={4} xs={12} className={classes.headerLeftContainer}>
+            <Grid
+              container
+              display="flex"
+              direction="column"
+              align="center"
+              justify="space-evenly"
+              className={classes.cardHeadLeft}
+            >
+              <Avatar
+                src={profile_pic}
+                className={classes.cardHeadAvatar}
+                borderColor="white"
+              />
+              <Typography variant="h6" className={classes.cardHeadName}>
+                Christine Wilconson
+              </Typography>
+              <Typography className={classes.cardHeadLocation}>
+                Ontario, Canada
+              </Typography>
+              <Button className={classes.cardSendMessage}>Send Message</Button>
+            </Grid>
+          </Grid>
+
+          <Grid item lg={8} xs={12} className={classes.headerRightContainer}>
+            {/* heading */}
+            <Grid container xs={12} className={classes.cardHeadRight}>
+              <Grid className={classes.bioContainer} xs={8}>
+                <Typography
+                  component="h6"
+                  xs={12}
+                  md={4}
+                  className={classes.cardBio}
+                >
+                  <Box paddingBottom="1em" fontWeight="fontWeightBold">
+                    ABOUT ME:
+                  </Box>
+                  <Typography className={classes.bioHead}>
+                    {bioTitle}
+                  </Typography>
+                  <Typography className={classes.biography}>{bio}</Typography>
+                </Typography>
+                <Grid className={classes.cardFavorites}>
+                  <Box paddingBottom="0.75em" fontWeight="fontWeightBold">
+                    FAVORITE CUISINE:
+                  </Box>
+                  <List xs={12} md={4} className={classes.favoritesList}>
+                    <ListItem className={classes.favoriteCuisine}>
+                      <ListItemText className={classes.cuisineText}>
+                        Mediterranean
+                      </ListItemText>
+                    </ListItem>
+                    <ListItem className={classes.favoriteCuisine}>
+                      <ListItemText className={classes.cuisineText}>
+                        Chinese
+                      </ListItemText>
```

Comment:
It's okay to not use Typography here because `ListItemText` already wraps around a `Typography`. That also means you can pass it variant/color props instead of using a style. When you can, use a prop instead of styling. This will allow more uniformity across the app

Reply:
Oh cool! Thank you, I'll change that :D

Reply:
> It's okay to not use Typography here because `ListItemText` already wraps around a `Typography`. That also means you can pass it variant/color props instead of using a style. When you can, use a prop instead of styling. This will allow more uniformity across the app

Okay, I've been trying but I have no idea what you mean. I've tried passing "color="white"" to the ListItemText component which yielded nothing. but using style={{ color:"white" }} worked but you're saying to use props instead of styling. is style a prop or styling?

Reply:
I couldn't change to using props instead of regular styles because trying to do that required disabling the built in Typography component then passing it a style={{ <style here> }} prop which defeats the purpose of using props. 
If I'm wrong, I'd love to read some documents that tell me so and how to use props instead of style but for now, I'm going to leave this as it is.